### ISSUE TITLE
Type argument in android bridge with SDK 4.12

### ISSUE THREAD

**Issue by vidangelci** [THEOplayer/react-native-theoplayer#90]
It appears that the latest Android SDK release (`4.12`) breaks react-native-theoplayer. We just started getting this error when trying to build for react-native Android. 

```
e:  /home/projects/playerapp/node_modules/react-native-theoplayer/android/src/main/java/com/theoplayer/PlayerEventEmitter.kt: (497, 96): Type argument is not within its bounds: should be subtype of 'QualityChangedEvent<Q!, ActiveQualityChangedEvent!>!'
e: /home/projects/playerapp/node_modules/react-native-theoplayer/android/src/main/java/com/theoplayer/PlayerEventEmitter.kt: (499, 115): Type argument is not within its bounds: should be subtype of 'QualityChangedEvent<Q!, ActiveQualityChangedEvent!>!'
e: /home/projects/playerapp/node_modules/react-native-theoplayer/android/src/main/java/com/theoplayer/PlayerEventEmitter.kt: (502, 121): Type argument is not within its bounds: should be subtype of 'QualityChangedEvent<Q!, ActiveQualityChangedEvent!>!'
```

Pinning any earlier version of the SDK with (for example):

`THEOplayer_sdk = "4.11.0"`

works fine; however, since that's not the default, the backwards incompatibility means that `4.12` will likely be grabbed by existing CI/CD pipelines and fail.

Please do let us know when there's a version of react-native-theoplayer that works with SDK `4.12`.

**Comment by danychi** [comment]
+1 having the same issue 😟

**Comment by tvanlaerhoven** [comment]
@vidangelci @danychi thanks for flagging this!
 
We released a 1.8.2 with a fix for the event cast.
